### PR TITLE
OCD-3923 UI: Compare & Table Enhancements

### PR DESCRIPTION
**Pull request by mattystank**

### CODE REVIEW

File: src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React, { useEffect, useState } from 'react';
 import { func, object } from 'prop-types';
 import {
-  Button,
+  IconButton,
```
Comment (andlar) [inline src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx]: Can you sort these imports? It's helpful having the items in the import objects sorted alphabetically

File: src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx [hunk 2]
```diff
@@ -59,105 +91,98 @@ function ChplCompareUploadedAndSystemDevelopers(props) {
...
-                <TableCell scope="row">Developer Name</TableCell>
+                <TableCell className={classes.sectionCell} scope="row">Developer Name</TableCell>
                 <TableCell>{ system.name }</TableCell>
                 <TableCell>{ uploaded.name }</TableCell>
               </TableRow>
               <TableRow
                 className={(system.website !== uploaded.website) ? classes.differentValue : ''}
               >
-                <TableCell scope="row">Website</TableCell>
+                <TableCell className={classes.sectionCell} scope="row"><Typography><b>Website</b></Typography></TableCell>
```
Comment (andlar) [inline src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx]: Why does this one have the Typography & <b> tags, while the others don't? I'd expect it to be standardized

File: src/app/components/listing/confirm/confirm-developer.jsx [hunk 3]
```diff
@@ -351,15 +359,17 @@ function ChplConfirmDeveloper(props) {
...
-                          Compare with uploaded information
-                        </Button>
-                      </CardActions>
+                    <Button
+                      variant="contained"
+                      color="secondary"
+                      onClick={() => setIsShowingComparison(true)}
+                    >
+                      Compare with uploaded information
+                      <CompareArrowsIcon className={classes.iconSpacing} />
```
Comment (andlar) [inline src/app/components/listing/confirm/confirm-developer.jsx:370]: It occurs to me that we might not need the `classes.iconSpacing` thing, if the only thing we're trying to do is put a space between the text and the icon. Can you see if this construct works, and if so, maybe we can stop using this class?

```suggestion
                      Compare with uploaded information
                      {' '}
                      <CompareArrowsIcon />
```

Comment (mattystank) [reply src/app/components/listing/confirm/confirm-developer.jsx:370]: The code you provide didnt create enough spacing for me.

File: src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx [hunk 4]
```diff
@@ -1,28 +1,48 @@
...
   CircularProgress,
+  IconButton,
   Table,
   TableBody,
   TableCell,
   TableContainer,
   TableHead,
   TableRow,
   makeStyles,
+  Typography,
```
Comment (andlar) [inline src/app/components/listing/confirm/compare-uploaded-and-system-developers.jsx]: Is this used?

File: src/app/components/listing/confirm/confirm-developer.jsx [hunk 5]
```diff
@@ -351,15 +359,17 @@ function ChplConfirmDeveloper(props) {
                       </MenuItem>
                     ))}
                   </ChplTextField>
+                  <br />
                   { !isShowingComparison
                     && (
-                      <CardActions>
```
Comment (andlar) [inline src/app/components/listing/confirm/confirm-developer.jsx:356]: I see the `CardActions` element is removed. Is it still being imported? Is there a reason not to put the button in a CardActions section?

Comment (mattystank) [reply src/app/components/listing/confirm/confirm-developer.jsx:356]: for card action element our theme puts a grey background on the card action container, i took it out so I wouldn't need to add another class to take out the grey background

Comment (andlar) [reply src/app/components/listing/confirm/confirm-developer.jsx:356]: Okay. Did you remove the import?

Comment (mattystank) [reply src/app/components/listing/confirm/confirm-developer.jsx:356]: Yes its below

Comment (andlar) [reply src/app/components/listing/confirm/confirm-developer.jsx:356]: You removed the use of it, down here around line 366. But the CardActions element is imported up at line 5 or so. If we're not using the CardActions any more, down here in the display, then we shouldn't be importing the component up at the top

**Pull request merged**